dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -234,3 +234,7 @@ scheme.incorrect.error.useraction=The scheme type must be http, https, ws, or ws
 upgradeHttpToWebSocket.failed=CWWKH0055E: The jakarta.websocket.server.ServerContainer#upgradeHttpToWebSocket call failed due to {0}. 
 upgradeHttpToWebSocket.failed.explanation=The server could not upgrade the HTTP or HTTPS request to the ws/wss protocol.
 upgradeHttpToWebSocket.failed.useraction=Review the error to identify why the request failed to upgrade.
+
+maxmessagesize.exceeded=CWWKH0056E: The maxMessageSize for method {0} exceeded Integer.MAX_VALUE for class {1}.
+maxmessagesize.exceeded.explanation=The OnMessage annotation parameter maxMessageSize can only be set to a maxmimum value of Integer.MAX_VALUE as per the WebSocket 2.2 specification.
```

Comment:
Suggested change:

The maxMessageSize OnMessage annotation parameter sets the maximum message size in bytes for a method. maxMessageSize can be set only to a maximum value of Integer.MAX_VALUE.

(I removed "as per the WebSocket 2.2 specification" because the spec allows users to specify `-1` for no maximum. The point seems to be that maxMessageSize can't be more than Integer.MAX_VALUE.)